code/modules/mob/living/carbon/xenomorph/castes/puppet/castedatum_puppet.dm

```diff
@@ -0,0 +1,24 @@
+/datum/xeno_caste/puppet
+	caste_name = "Puppet"
+	display_name = "Puppet"
+	upgrade_name = ""
+	caste_desc = "A grotesque puppet of a puppeteer."
+	wound_type = ""
+
+	caste_type_path = /mob/living/carbon/xenomorph/puppet
+
+	tier = XENO_TIER_MINION
+	upgrade = XENO_UPGRADE_BASETYPE
+	melee_damage = 15
```

Comment:
https://github.com/tgstation/TerraGov-Marine-Corps/assets/22431091/a80e25ca-a81f-49ff-a6aa-b47380b3c61b

Full HP instant chest frac in medium armor with not a thing I could do. The only way you keep slash damage this high is making the puppets block eachother or something because this is not alright.

Reply:
flay nerf please :)

Reply:
reduced the damage by like 15 brute until bowl figures out something